recipes/sire/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "sire" %}
+{% set version = "2021.1.0" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/michellab/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: c4b8c657498c62d2bdfb8d856280c22256463665143352d5e04ce8cb61f4df14
+  patches: conda_build.patch
+
+build:
+  number: {{ build }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - git
+    - jom  # [win]
+    - make  # [unix]
+  host:
+    - {{ blas_impl }}
```

Comment:
Do you need to build for all implementations? This is only for packages that use internal APIs of different implementations. See the docs.

Reply:
No, we actually don't need BLAS as a requirement. We fully isolate BLAS / LAPACK behind our own interface, which is disabled during this particular build. However, we do use GSL, which pulls in a version of BLAS. For some reason there seems to be a mismatch between the version of BLAS used at build and runtime when listing GSL as a host and run dependency. (See the MKL issue earlier.) This only appears to happen for certain builds, i.e. OS and Python version.